### DISCUSSION TITLE
Low values for `wait-duration-in-open-state` in circuit breaker won't work as expected?

### DISCUSSION THREAD

**Discussion by rodrigouchoa** [resilience4j/resilience4j#2160] [General]
Hi all! 

I'm using `resilience4j` to create a circuit breaker in a Spring Boot 3.2.4 application. Some of my integration tests will make sure circuit is working as expected. 

If I create a test to assert the circuit is transitioning from OPEN to HALF_OPEN, I obviously don't want the test to wait the same as my app would in production. 
So for the tests I set a low value for the property `resilience4j.circuitbreaker.instances.my-cb.wait-duration-in-open-state.seconds`, like 5 or 10 seconds. However it always takes almost a minute for the transition to happen. 

Is this a known limitation? Not being able to be accurate to seconds, only minutes?

**Comment by RobWin** [comment]
Hi,
can you show your test?

**Reply by rodrigouchoa** [reply]
Hi! 
I solved the problem. Looks as though `wait-duration-in-open-state.seconds: 10` property simply doesn't work. When I changed it to `waitDurationInOpenState: 10s` it worked as expected

**Reply by RobWin** [reply]
ok. thanks for your feedback
